From 88968ba474d993670b0b35d839ab2b7d9f575699 Mon Sep 17 00:00:00 2001
From: Andrew Udvare <audvare@gmail.com>
Date: Tue, 22 Nov 2022 07:34:03 -0500
Subject: [PATCH 1/3] Better first launch config defaults

- Disable file log
- Disable info level logging
- Disable stub (compare with Wine's stub logging) logging
- Disable warning level logging
- Disable Discord integration
- Disable checking updates on startup
---
 .../Configuration/ConfigurationState.cs              | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/Ryujinx.Ui.Common/Configuration/ConfigurationState.cs b/src/Ryujinx.Ui.Common/Configuration/ConfigurationState.cs
index b017d384..39b45a04 100644
--- a/src/Ryujinx.Ui.Common/Configuration/ConfigurationState.cs
+++ b/src/Ryujinx.Ui.Common/Configuration/ConfigurationState.cs
@@ -758,7 +758,7 @@ namespace Ryujinx.Ui.Common.Configuration
 
         public void LoadDefault()
         {
-            Logger.EnableFileLog.Value = true;
+            Logger.EnableFileLog.Value = false;
             Graphics.BackendThreading.Value = BackendThreading.Auto;
             Graphics.ResScale.Value = 1;
             Graphics.ResScaleCustom.Value = 1.0f;
@@ -768,9 +768,9 @@ namespace Ryujinx.Ui.Common.Configuration
             Graphics.PreferredGpu.Value = "";
             Graphics.ShadersDumpPath.Value = "";
             Logger.EnableDebug.Value = false;
-            Logger.EnableStub.Value = true;
-            Logger.EnableInfo.Value = true;
-            Logger.EnableWarn.Value = true;
+            Logger.EnableStub.Value = false;
+            Logger.EnableInfo.Value = false;
+            Logger.EnableWarn.Value = false;
             Logger.EnableError.Value = true;
             Logger.EnableTrace.Value = false;
             Logger.EnableGuest.Value = true;
@@ -782,8 +782,8 @@ namespace Ryujinx.Ui.Common.Configuration
             System.TimeZone.Value = "UTC";
             System.SystemTimeOffset.Value = 0;
             System.EnableDockedMode.Value = true;
-            EnableDiscordIntegration.Value = true;
-            CheckUpdatesOnStart.Value = true;
+            EnableDiscordIntegration.Value = false;
+            CheckUpdatesOnStart.Value = false;
             ShowConfirmExit.Value = true;
             HideCursor.Value = HideCursorMode.OnIdle;
             Graphics.EnableVsync.Value = true;
-- 
2.43.0

